server/auth/google.js

```diff
@@ -34,15 +34,11 @@ if (!process.env.GOOGLE_CLIENT_ID || !process.env.GOOGLE_CLIENT_SECRET) {
       const name = profile.displayName
       const email = profile.emails[0].value
 
-      User.find({where: {googleId}})
-        .then(
-          foundUser =>
-            foundUser
-              ? done(null, foundUser)
-              : User.create({name, email, googleId}).then(createdUser =>
-                  done(null, createdUser)
-                )
-        )
+      User.findOrCreate({
+        where: {googleId},
+        defaults: {name, email}
+      })
+        .then(([user]) => done(null, user))
         .catch(done)
```

Comment:
While we're here can we migrate this to `async/await`. I think the subtle inconsistencies are muddying the waters for some students.

```js
      try {
        const [user] = await User.findOrCreate({
          where: {googleId},
          defaults: {name, email}
        })
        done(null, user)
      }
      catch (error) {
        done(error);
      }
```

Reply:
Currently the codebase doesn't include any `async`/`await` except in the seed file (where it is announced with great fanfare). I think we should treat the conversion to `async`/`await` as a separate PR, perhaps resurrecting #20 (or redoing it from scratch). But I am 100% in favor of doing the conversion, I think we only delayed it because WDF is still on the original curriculum.

Reply:
@glebec WDF is using Async/Await and this would be fine!

Reply:
Yeah, don't need to block this PR for that and the capstones are beginning.

Reply:
Thanks @coreygreenwald. In that case I am fine to either update this PR or merge it now and start working on bringing #20 up to speed / redoing it.

Reply:
My approval stands with or without `async/await`on this PR. Either way we need a larger sweep for `.then`s.